File: examples/python/models/Circle.py
import math
from .ShapeInterface import ShapeInterface
from .ManageShapeInterface import ManageShapeInterface

class Circle(ShapeInterface, ManageShapeInterface):
    def __init__(self, radius: int):
        self.radius = radius

    def area(self) -> float:
        return math.pi * (self.radius ** 2)

    def calculate(self) -> float:
        return self.area()

File: examples/csharp/Model/Circle.cs
using System;

namespace Model
{
    public class Circle: IShape, IManageShape
    {
        public int Radius { get; private set; }

        public Circle(int radius)
        {
            this.Radius = radius;
        }

        public double Area()
        {
            return Math.PI * Math.Pow(this.Radius, 2);
        }

        public double Calculate()
        {
            return this.Area();
        }
    }
}
